NMS-14131: DCB: UI Documentation (#4544)
OpenNMS/opennms: docs/modules/reference/pages/service-assurance/monitors/DeviceConfigMonitor.adoc

| schedule
| Schedule to fetch configuration, in the form of cron expression.

> | 0 0 0 * * ?

Reviewer: format to keep the cron expression separate from the following note
```suggestion
| `0 0 0 * * ?`
```

== Schedule and interval

The service uses an interval of 300 seconds and a cron expression for the schedule.

> The default cron expression runs every 24 hrs at 12:00 a.m. (0 0 0 * * ?).

Reviewer: ```suggestion
The default cron expression runs every 24 hrs at 12:00 a.m. (`0 0 0 * * ?`).
```

The monitor checks every 300 seconds (interval) for the next trigger time in the cron schedule.
It retrieves the configuration only when the next trigger time matches the current time (current time must be greater than the trigger time).

> For example, if the monitor checks at 11:59, but the trigger time is 12:00 a.m., the monitor retrieves the configuration at 12:04 a.m., which is the next check interval.

Reviewer: ```suggestion
For example, if the monitor checks at 11:59 p.m., but the trigger time is 12:00 a.m., the monitor will retrieve the configuration at the next check interval at 12:04 a.m.
```

The Device Config Monitor service uses an interval of 300secs and a cron expression for the schedule.
The default cron expression runs every 24 hrs at 12.00am (0 0 0 * * ?).
The monitor retrieves the device configuration when the next trigger from the schedule falls within the interval.

> ##Chandra, could you update the text in line 57?##

Author: Yes, I believe that is what is happening, but I realized I didn't fully understand it when I tried to document it. Hoping Chandra can shed some light.
